docs/en/Deployment.md

```diff
@@ -0,0 +1,145 @@
+# Operation and maintenance management
+
+## Cluster building
+
+### 1 Single Master mode
+
+This is the simplest, but also the riskiest, mode that makes the entire service unavailable once the Broker restarts or goes down.Online environments are not recommended, but can be used for local testing and development.Here are the steps to build。
+
+**1）Start NameServer**
+
+```shell
+### Start Name Server first
+$ nohup sh mqnamesrv &
+ 
+### Then verify that the Name Server starts successfully
+$ tail -f ~/logs/rocketmqlogs/namesrv.log
+The Name Server boot success...
+```
+
+We can see 'The Name Server boot success.. ' in namesrv.log That indicates that the start NameServer has been successful.
+
+**2）Start Broker**
+
+```shell
+### Also start Broker first
+$ nohup sh bin/mqbroker -n localhost:9876 &
+
+### Then verify that the Broker is started successfully, for example, the IP of Broker is 192.168.1.2 and the name is broker-a
+$ tail -f ~/logs/rocketmqlogs/Broker.log 
+The broker[broker-a, 192.169.1.2:10911] boot success...
+```
+
+We can see 'The broker[brokerName,ip:port] boot success.. ' in Broker.log That indicates that the start Broker has been successful.
+
+### 2 Multiple Master mode
+
+The advantages and disadvantages of a cluster without Slave, being all Master, such as 2 Master or 3 Master, are as follows:
+
+- Advantages: simple configuration and a single Master outage or restart maintenance in the cluster has no impact on the application.When the disk is configured as RAID10, messages are not lost because the RAID10 disk is very reliable, even if the machine is not recoverable (In the case of asynchronous brush disk mode of the message, a small number of messages are lost;If the brush mode of a message is synchronous, a single message is not lost).In this mode, the performance is the highest.
```

Comment:
Is it better to use 'flush disk' replace  'brush disk'

Reply:
numbered list is recommended here, as below:
1. simple configuration.
2. Outage or restart(for maintenance) of just one master node has no impact on the application.
3. XXXXXX
4. XXXXXX

and a better expression for "a single message is not lost" is "no message will be lost".